pulsar-proxy/src/main/java/org/apache/pulsar/proxy/server/ProxyConnection.java

```diff
@@ -119,6 +123,9 @@ public ProxyConnection(ProxyService proxyService, Supplier<SslHandler> sslHandle
         this.service = proxyService;
         this.state = State.Init;
         this.sslHandlerSupplier = sslHandlerSupplier;
+        if (proxyService.getConfiguration().isEnableShareTimer()) {
```

Comment:
I am not sure we need to make this implementation configurable.
it is a good thing to share this timer.